starter/script.js

```diff
@@ -1 +1,119 @@
-// put code here!
+console.log("JS connected")
+
+//thoughts about this: should I use jquery or really stick with vanilla?
+// could use a combo.
+//whats the learning objective here?
```

Comment:
Your call.

Although this exercise isn't mean to involve DOM manipulation so no need for jQuery here (unless you make a use for it).